src/components/Viewport.ts

```diff
@@ -1741,6 +1742,25 @@ export default class Viewport {
         panel: siblingPanel,
         direction,
         range: indexRange,
+        fill: (element: ElementLike | ElementLike[]) => {
+          const panelManager = this.panelManager;
+          if (!siblingPanel) {
+            return this.insert(panelManager.getRange().max + 1, element);
+          }
+
+          const parsedElements = parseElement(element);
```

Comment:
@WoodNeck 
It's hard to read because there are too many `?`. I think it's better to change it to `if` phrase.

Reply:
I agree that can be burdensome to read. I'll fix that.